warehouse/scripts/templates/ci_report.md

```diff
@@ -0,0 +1,39 @@
+Warehouse report 📦
+
+{% if new_models %}
+### New models 🌱
+
+{% for model in new_models %}
+{{ model }}
+{% endfor %}
+{% endif %}
+
+{% if changed_incremental_models %}
+### Changed incremental models 🔀
+
+{% for model in changed_incremental_models %}
+{{ model }}
+{% endfor %}
+{% endif %}
+
+### DAG
+Legend (in order of precedence)
+
+| Resource type                                  | Indicator | Resolution                            |
+|------------------------------------------------|-----------|---------------------------------------|
+| Large table-materialized model                 | Red       | Make the model incremental            |
+| Large model without partitioning or clustering | Orange    | Add partitioning and/or clustering    |
+| View with more than one child                  | Yellow    | Materialize as a table or incremental |
+| Incremental                                    | Blue      |                                       |
```

Comment:
I am confused about the blue vs. green when the main color I've actually seen is "aquamarine"